spec/request/preneeds/attachment_types_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe 'Preneeds Attachment Types Integration', type: :request do
   include SchemaMatchers
+  before(:each) { Redis.current.flushall }
```

Comment:
Same comment as before, I think if this is tested at the model level, and stubbed at the request spec level, you won't need to do before(:each) like this -- which slows down specs unnecessarily.